### PR TITLE
Feanil/fix description

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,6 +14,12 @@ commands =
     pytest {posargs}
 
 [testenv:quality]
+whitelist_externals =
+    bash
 commands =
     pycodestyle --config=.pep8 edx_rest_api_client
     pylint --rcfile=pylintrc edx_rest_api_client
+    # Ideally we could say `twine check {env:TOX_PACKAGE}` but this throws
```
[the remark this answers is not in the document]
Comment (jmbowman) [reply tox.ini]: I actually meant `passenv`, but both are ways of dealing with environment variables in tox:

* https://tox.readthedocs.io/en/latest/config.html#conf-passenv
* https://tox.readthedocs.io/en/latest/config.html#conf-setenv

[the remark this answers is not in the document]
Comment (feanil) [reply tox.ini]: Unfortunately beacuse this environment variable is generated by tox, it's just not available early enough for me to make use of it.  I found a different var that sholud be sufficient. `{distdir}`

File: requirements/base.txt [hunk 2]
```diff
@@ -4,13 +4,14 @@
 #
 #    make upgrade
 #
+asgiref==3.2.4            # via django
 certifi==2019.11.28       # via requests
 chardet==3.0.4            # via requests
 django-waffle==0.20.0     # via edx-django-utils
-django==2.2.11            # via edx-django-utils
+django==3.0.4             # via edx-django-utils
```
Comment (jmbowman) [inline requirements/base.txt]: Do we really want dev environments to get Django 3 by default already?

File: requirements/test.in [hunk 3]
```diff
@@ -11,3 +11,4 @@ pycodestyle
 pytest-cov                # pytest extension for code coverage statistics
 pytest-django             # pytest extension for better Django support
 responses
+twine
```
Comment (jmbowman) [inline requirements/test.in]: Could you add a comment explaining what we use it for?

File: requirements/base.txt (hunk 2 above)
Comment (feanil) [reply requirements/base.txt]: Tox is still running for 111 only right now I can add a new env for this? to test both.

File: requirements/test.in (hunk 3 above)
Comment (feanil) [reply requirements/test.in]: will do.

**Comment by feanil** [comment]
Created https://openedx.atlassian.net/browse/BOM-1397 for the follow up in the library cookiecutter

**Pull request merged**